qcodes_contrib_drivers/drivers/Bilt/ITest.py

```diff
@@ -0,0 +1,467 @@
+        elif isinstance(pos_sat,str):
+            self.write(chan_id + 'VOLT:SAT:POS MAX')
+
+
+    def _set_chan_neg_sat(self, chan:int,
+                                neg_sat: Union[int, str]) -> None:
+        chan_id = self.chan_to_id(chan)
+        if isinstance(neg_sat,(int,float)):
+            self.write(chan_id + 'VOLT:SAT:NEG {:.8f}'.format(neg_sat))
+        elif isinstance(neg_sat,str):
+            self.write(chan_id + 'VOLT:SAT:NEG MIN')
+
+
+    def _get_chan_pos_sat(self, chan:int) -> str:
+        chan_id = self.chan_to_id(chan)
+        return self.ask(chan_id + 'VOLT:SAT:POS ?')
+
+
+    def _get_chan_neg_sat(self, chan:int) -> str:
+        chan_id = self.chan_to_id(chan)
+        return self.ask(chan_id + 'VOLT:SAT:NEG ?')
+
+
+    def _get_chan_state(self, chan:int) -> bool:
+        """
+        Get channel power state
+        
+        Args:
+            chan: The 1-indexed channel number
+            
+        Returns:
+            state: Power state 
+        """
+        chan_id = self.chan_to_id(chan)
+        state = self.ask(chan_id + 'OUTP ?')
+        
+        if state == '1':
+            return True
+        elif state == '0':
+            return False
+
+
+    def _set_chan_state(self, chan:int,
+                              state: bool) -> None:
+        """
+        Set channel power state
+        
+        Args:
+            chan: The 1-indexed channel number
+            state: power state
+        """
+        chan_id = self.chan_to_id(chan)
+        if state:
+            state_str = 'on'
+        else:
+            state_str = 'off'
+        self.write(chan_id + 'OUTP ' + state_str)
+
+
+    def _set_chan_name(self, chan:int,
+                             name: str) -> None:
+        """
+        Set the name of the channel
+
+        Args:
+            chan: Channel to be named
+            name: Name of the channel
+        """
+        chan_id = self.chan_to_id(chan)
+        self.write(chan_id + 'chan:name "{}"'.format(name))
+
+
+    def chan_to_ic(self, chan:int) -> Tuple[int]:
+        """
+        Indexing conversion from channel number (1 to 16)
+        to iX;c;
+        Args:
+            chan: The 1-indexed channel number
+            
+        Returns:
+            i,c: i=card number, c=channel number of card i
+        """
+        i = ceil(chan/4.)
+        c = chan-(i-1)*4
+        return i,c
+
+
+    def chan_to_id(self, chan:int) -> str:
+        i,c = self.chan_to_ic(chan)
+
+        return 'i{};c{};'.format(i,c)
+
+
+    def _set_chan_synchronous(self, chan: int,
+                                    synchronous: bool) -> None:
+        """
+        Set channel synchronicity.
+
+        Args:
+            chan: The 1-indexed channel number
+            synchronous: If the channel is synchronous.
+        """
+        
+        chan_id = self.chan_to_id(chan)
+        self._synchronous[chan_id] = synchronous
+
+
+    def _get_chan_synchronous(self, chan: int) -> None:
```

Comment:
You are returning self._synchronous[chan_id], but saying return is None.  Please fix ths

Reply:
Corrected.